pi-apps-terminal-bash-edition.sh

```diff
@@ -3,29 +3,19 @@
 ### ignore some shellcheck warnings
 # shellcheck disable=SC2145,SC2199,SC2068
 
-#determine if host system is 64 bit arm64 or 32 bit armhf
-if [ "$(od -An -t x1 -j 4 -N 1 "$(readlink -f /sbin/init)")" = ' 02' ];then
-  arch=64
-elif [ "$(od -An -t x1 -j 4 -N 1 "$(readlink -f /sbin/init)")" = ' 01' ];then
-  arch=32
-else
-  echo -e "\e[1mFailed to detect OS CPU architecture! Something is very wrong.\e[0m"
-fi
-
 #directory variables
-PI_APPS_DIR="$HOME/pi-apps"
+if [ -z "$DIRECTORY" ]; then
+	DIRECTORY="$HOME/pi-apps"
+fi
 
 #check if '~/pi-apps/api' exists
-if [[ ! -f "$HOME/pi-apps/api" ]]; then
+if [ ! -f "$DIRECTORY/api" ]; then
```

Comment:
```suggestion
if [[ ! -f "$DIRECTORY/api" ]]; then
```
Doesn't really matter here.
but generally, `[[` is the more advanced version of `[`/`test`, so I prefer to use it.

Reply:
Good link, thanks!